articles/api/authentication/_logout.md

```diff
@@ -24,50 +32,62 @@ curl --request GET \
   });
   
   webAuth.logout({
-    returnTo: 'YOUR_LOGOUT_URL',
-    client_id: '${account.clientId}'
+    post_logout_redirect_uri: 'YOUR_LOGOUT_URL',
+    id_token_hint: 'YOUR_ID_TOKEN_HINT',
+    logout_hint: 'YOUR_LOGOUT_HINT'
   });
 </script>
 ```
 
 <%= include('../../_includes/_http-method', {
   "http_badge": "badge-primary",
   "http_method": "GET",
-  "path": "/v2/logout",
+  "path": "/oidc/logout",
   "link": "#logout"
 }) %>
 
-Use this endpoint to logout a user. If you want to navigate the user to a specific URL after the logout, set that URL at the `returnTo` parameter. The URL should be included in any the appropriate `Allowed Logout URLs` list:
-- If the `client_id` parameter is included, the `returnTo` URL must be listed in the `Allowed Logout URLs` set at the application level (see [Setting Allowed Logout URLs at the App Level](/logout#set-the-allowed-logout-urls-at-the-application-level)).
-- If the `client_id` parameter is NOT included, the `returnTo` URL must be listed in the `Allowed Logout URLs` set at the tenant level (see [Setting Allowed Logout URLs at the Tenant Level](/logout#set-the-allowed-logout-urls-at-the-tenant-level)).
-- If the `client_id` parameter is included and the `returnTo` URL is NOT set, the server returns the user to the first Allowed Logout URLs set in the Dashboard (see [Setting Allowed Logout URLs at the Tenant Level](/logout#set-the-allowed-logout-urls-at-the-tenant-level)).
+Use this endpoint to logout a user. If you want to navigate the user to a specific URL after the logout, set that URL at the `post_logout_redirect_uri` parameter. The URL should be included in any the appropriate `Allowed Logout URLs` list:
+
+- If the `client_id` parameter is included, the `post_logout_redirect_uri` URL must be listed in the `Allowed Logout URLs` set at the application level (see [Setting Allowed Logout URLs at the App Level](https://auth0.com/docs/logout#set-the-allowed-logout-urls-at-the-application-level)).
```

Comment:
URLs in this repo only use the ending slug, so what's after /docs: [Setting Allowed Logout URLs at the App Level](/logout#set-the-allowed-logout-urls-at-the-application-level)

Reply:
Same for the URLs below.

Reply:
I've changed this section a bit to try and cover these different cases. I fear that it's getting quite complicated and that there isn't a clear understanding, at least in these docs, what overrides what. Looking for suggestions to make this better!